decidim-core/db/migrate/20200930125538_move_cta_to_hero_content_block.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class MoveCtaToHeroContentBlock < ActiveRecord::Migration[5.2]
+  class Organization < ApplicationRecord
+    self.table_name = :decidim_organizations
+  end
+
+  def change
+    Decidim::ContentBlock.reset_column_information
+    Organization.find_each do |organization|
+      content_block = Decidim::ContentBlock.find_by(organization: organization, scope_name: :homepage, manifest_name: :hero)
+      settings = {}
+      cta_button_text = organization.cta_button_text || {}
+      settings = cta_button_text.inject(settings) { |acc, (k, v)| acc.update("cta_button_text_#{k}" => v) }
+
+      unless organization.cta_button_path.empty?
+        # Adds i18n support to cta_button_path for every defined lang in cta_button_text
+        settings = cta_button_text.inject(settings) { |acc, (k, _v)| acc.update("cta_button_path_#{k}" => organization.cta_button_path) }
```

Comment:
It's a little hacky for converting from a regular string to a i18n string. Maybe there's an already common convention in Decidim for doing this?

Reply:
None comes to my mind, the only similar is a specs utility that can be found in `decidim-core/lib/decidim/faker/localized.rb#localized`